apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v1/eobByPatientIdAll.json

```diff
@@ -2041,7 +2053,8 @@
         "diagnosisCodeableConcept" : {
           "coding" : [ {
             "system" : "http://hl7.org/fhir/sid/icd-10",
-            "code" : "R4444"
+            "code" : "A05",
+            "display" : "\"OTH BACTERIAL FOODBORNE INTOXICATIONS, NEC\""
```

Comment:
I'm sure we get these descriptions from external sources, but WTF....why they feel compelled to include quotes within a description is beyond me.